content/en/docs/ops/configuration/traffic-management/manage-mesh-certificates/index.md

```diff
@@ -0,0 +1,195 @@
+[pilot-discovery](/docs/reference/commands/pilot-discovery/), this document will only
+outline some of them.
+{{< /tip >}}
+
+{{< tabset category-name="certificates" >}}
+
+{{< tab name="IstioOperator" category-value="iop" >}}
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    pilot:
+      env:
+        CITADEL_SELF_SIGNED_CA_RSA_KEY_SIZE: 4096
+        <key>: <value>
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< tab name="Helm" category-value="helm" >}}
+
+{{< text yaml >}}
+pilot:
+  env:
+    CITADEL_SELF_SIGNED_CA_RSA_KEY_SIZE: 4096
+    <key>: <value>
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+### Self-signed CA
+
+To specify the signature algorithm for istiod, you need to set the
+`CITADEL_SELF_SIGNED_CA_CERT_ALGORITHM` environment variable. The available options are `RSA`
+(default, no need to set explicitly) or `ECC`.
+
+By configuring the following environment variables, you can manage other properties of this certificate based on its type:
+
+- `CITADEL_SELF_SIGNED_CA_RSA_KEY_SIZE`
+- `ECC_CURVE`
+- `ECC_SIGNATURE_ALGORITHM`
+
+These variables allow you to fine-tune and control various aspects of the certificate management process.
+
+### istiod Certificate
+
+To control the type of certificate generated by istiod for itself, you can set
+`ECC_SIGNATURE_ALGORITHM` to `ECDSA`.
+
+It's strength can be configured using either `ECC_CURVE` or `WORKLOAD_RSA_KEY_SIZE`.
+
+## Sidecars
+
+Since sidecars manage their own certificates for in-mesh communication, the sidecars
+are responsible for managing their private keys and generated CSRs. The sidecar
+injector needs to be modified to inject the environment variables to be used for
+this purpose.
+
+{{< tip >}}
+While there are many environment variables that can be changed for
+[pilot-agent](/docs/reference/commands/pilot-agent/), this document will only
+outline some of them.
+{{< /tip >}}
+
+{{< tabset category-name="gateway-install-type" >}}
+
+{{< tab name="IstioOperator" category-value="iop" >}}
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    defaultConfig:
+      proxyMetadata:
+        CITADEL_SELF_SIGNED_CA_RSA_KEY_SIZE: 4096
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< tab name="Helm" category-value="helm" >}}
+
+{{< text yaml >}}
+meshConfig:
+  defaultConfig:
+    proxyMetadata:
+      CITADEL_SELF_SIGNED_CA_RSA_KEY_SIZE: 4096
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< tab name="Annotation" category-value="annotation" >}}
+
+{{< text yaml >}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sleep
+spec:
+  ...
+  template:
+    metadata:
+      ...
+      annotations:
+        ...
+        proxy.istio.io/config: |
+          CITADEL_SELF_SIGNED_CA_RSA_KEY_SIZE: 4096
+    spec:
+      ...
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+### Signature Algorithm
+
+By default, the sidecars will create RSA-based certificates. If you want to change it to
```

Comment:
I think this line needs `RSA-based` added to `.spelling`, and that just makes me angry.